### PR TITLE
Post, comments, like post

### PR DESCRIPTION
**Pull request by AnuragSahu11**
*Create new post , saves post to firebase
*Like , unlike post
*Comment on post
Site will not work with forced https, ie Brave browser

### CODE REVIEW

**Review by GuptaEesh — COMMENTED** [review]
Everything works good, good that you used firebase.
Good to merge

File: src/App.css [hunk 1]
```diff
@@ -1,9 +1,10 @@
 .App {
-  height: 100vh;
+  /* height: 100vh; */
```
Comment (saadahmadhere) [inline src/App.css]: remove unwanted comments

File: src/components/post/post-container.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  margin-right: 10px;
+}
+.comment_row {
+  margin-top: 8px;
+}
+.add_comment_button {
+  margin-left: 8px;
+}
+
+@media only screen and (max-width: 1600px) {
```
Comment (saadahmadhere) [inline src/components/post/post-container.css:32]: nice use of media queries, it encloses almost every screen size

File: src/components/post/post.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+import { LikeTwoTone, LikeOutlined, CommentOutlined } from "@ant-design/icons";
+import { useState } from "react";
+import { Comments } from "./comments";
+
+const Post = ({ postData }) => {
+  const [showComments, setShowComments] = useState(false);
+  const { Meta } = Card;
+  const { Text } = Typography;
+
+  //   const { title, content, time, likes, postBy } = postData;
```
Comment (saadahmadhere) [inline src/components/post/post.jsx]: remove comment

**Review by saadahmadhere — APPROVED** [review]
- Clean code.
- Well written media queries.
- Proper use of dispatch function.

File: src/App.css (hunk 1 above)
Comment (AnuragSahu11) [reply src/App.css]: Yup, thanks

File: src/components/post/post-container.css (hunk 2 above)
Comment (AnuragSahu11) [reply src/components/post/post-container.css:32]: Thanks

File: src/components/post/post.jsx (hunk 3 above)
Comment (AnuragSahu11) [reply src/components/post/post.jsx]: yes will remove it

**Pull request merged**